### PR TITLE
Added front door config for nfdiv civil partnership

### PR DESCRIPTION
**Pull request by nitinprabhu**
- Use rules for disabling similar to what is configured for divorce 1.
- Added IP whitelisting

### CODE REVIEW

File: environments/prod/prod.tfvars [hunk 1]
```diff
@@ -328,6 +328,102 @@ frontends = [
       },
     ]
   },
+  {
+    product          = "nfdiv"
+    name             = "nfdiv-civil-partnership"
+    mode             = "Prevention"
```
Comment (linusnorton) [inline environments/prod/prod.tfvars:334]: Detection?

File: environments/prod/prod.tfvars [hunk 2]
```diff
@@ -328,6 +328,102 @@ frontends = [
...
+      },
+      {
+        match_variable = "RequestCookieNames"
+        operator       = "Equals"
+        selector       = "connect.sid"
+      },
+      {
+        match_variable = "RequestCookieNames"
+        operator       = "Equals"
+        selector       = "__auth-token"
```
Comment (linusnorton) [inline environments/prod/prod.tfvars:403]: nfdiv-session

File: environments/prod/prod.tfvars [hunk 3]
```diff
@@ -328,6 +328,102 @@ frontends = [
...
+    global_exclusions = [
+      {
+        match_variable = "RequestCookieNames"
+        operator       = "Equals"
+        selector       = "dtSa"
+      },
+      {
+        match_variable = "RequestCookieNames"
+        operator       = "Equals"
+        selector       = "cookies_policy"
```
Comment (linusnorton) [inline environments/prod/prod.tfvars:393]: nfdiv-cookie-preferences

File: environments/prod/prod.tfvars [hunk 4]
```diff
@@ -328,6 +328,102 @@ frontends = [
...
+      },
+      {
+        match_variable = "QueryStringArgNames"
+        operator       = "Equals"
+        selector       = "iss"
+      },
+      {
+        match_variable = "QueryStringArgNames"
+        operator       = "Equals"
+        selector       = "__auth-token"
```
Comment (linusnorton) [inline environments/prod/prod.tfvars:413]: kill

File: environments/prod/prod.tfvars [hunk 5]
```diff
@@ -328,6 +328,102 @@ frontends = [
...
+      },
+      {
+        match_variable = "QueryStringArgNames"
+        operator       = "Equals"
+        selector       = "_csrf"
```
Comment (linusnorton) [inline environments/prod/prod.tfvars:418]: kill

**Review by linusnorton — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**